2-factor-authentication/test/helper/jest.setup.ts

```diff
@@ -16,34 +16,28 @@ beforeAll(async () => {
      * Select the display you want to run your tests on, display 0 is your main display;
      * ignore if you have only one display
      */
-    display: 2,
-    port: 7007
+    //display: 1, enable it only when you have more then one display
+    port: 7007,
+
   });
 
   await uiController.start();
 
   aui = await UiControlClient.build({
-    credentials: {
-      workspaceId: '<your workspace id>',
-      token: '<your access token>',
-    },
-    uiControllerUrl:'http://127.0.0.1:7007'
+    uiControllerUrl: 'http://127.0.0.1:7007'
   });
 
   auiAndroid = await UiControlClient.build({
-    credentials: {
-      workspaceId: '<your workspace id>',
-      token: '<your access token>',
-    },
-    uiControllerUrl:'http://127.0.0.1:6769'
+    uiControllerUrl: 'http://127.0.0.1:6769',
+    'annotationLevel': AnnotationLevel.ON_FAILURE
   });
 
   await aui.connect();
   await auiAndroid.connect();
 });
 
 afterAll(async () => {
-  await uiController.stop();
+  //await uiController.stop();
```

Comment:
I would either 

remove start and stop or 

keep both (and add the missing start and stop for the second ui controller)

Reply:
we don't support android, so we can't start it automatically

Reply:
then, remove both ;) but you can, of course, it is just a little more code (executing stuff on the command line etc.)